extendedset/src/test/java/io/druid/extendedset/intset/ImmutableConciseSetIntersectionTest.java

```diff
@@ -0,0 +1,594 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.extendedset.intset;
+
+import com.google.common.collect.Lists;
+import junit.framework.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@RunWith(Parameterized.class)
+public class ImmutableConciseSetIntersectionTest
```

Comment:
Originally, all tests except intersection tests don't have to be parameterized because they run the same tests regardless of the parameter.
So, in this patch, Intersection tests are extracted to a separate ```ImmutableConciseSetIntersectionTest``` and only this class runs with parameters. This will especially reduce the testing time because ```testComplement9()``` which takes most of the testing time in ```ImmutableConciseSetTest``` runs only one time.